aiperf/exporters/gpu_telemetry_console_exporter.py

```diff
@@ -120,9 +120,9 @@ def _create_summary_header(self, table_title_base: str) -> str:
         """
         title_lines = ["NVIDIA AIPerf | GPU Telemetry Summary"]
 
-        endpoints_tested = self._telemetry_results.endpoints_tested
+        endpoints_configured = self._telemetry_results.endpoints_configured
```

Comment:
do we do any de-duplication if the user specifies an en endpoint that is already a default?